File: Networking/src/main/java/request/CreateParticipantRequest.java
package request;

import org.example.Participant;

public class CreateParticipantRequest implements Request{
    private final Participant participant;

    public CreateParticipantRequest(Participant participant) {
        this.participant = participant;
    }

    public Participant getParticipant() {
        return participant;
    }
}

File: Networking/Request/CreateParticipantRequest.cs
using System.Text.Json.Serialization;
using mpp_proiect_csharp_DianaGliga11.Model;

namespace Networking.Request
{
    public class CreateParticipantRequest : IRequest
    {
        [JsonPropertyName("participants")]
        public List<Participant> Participants{get;set;}

        [JsonConstructor]
        public CreateParticipantRequest()
        {
        }

        public CreateParticipantRequest(List<Participant> participants)
        {
            this.Participants = participants;
        }
    }
}